src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+3.1.0
+-----
+
+ * added shortcuts for redirected and forwarded request's profiles in the
```

Comment:
What about "added information about redirected and forwared requests to the profiler"?